tensorflow/contrib/makefile/samples/build_and_run_inception_hexagon.sh

```diff
@@ -219,7 +219,7 @@ if [[ "${BUILD_ONLY}" != "true" ]]; then
     adb push "${GEN_LIBS_DIR}/libhexagon_nn_skel.so" "/vendor/lib/rfsa/adsp"
 
     adb push -p \
-        "${TF_ROOT_DIR}/tensorflow/contrib/makefile/gen/bin/hexagon_graph_execution" \
+        "${TF_ROOT_DIR}/tensorflow/contrib/makefile/gen/bin/android_armeabi-v7a/hexagon_graph_execution" \
```

Comment:
It is possible this path is dependent on what target architecture you are building for.
@petewarden @andrewharp do you know if this value here is safe?